CHANGELOG.md

```diff
@@ -1,3 +1,88 @@
+# [MongooseIM 6.3.0](https://github.com/esl/MongooseIM/releases/tag/6.3.0) - 2024-10-XX
+
+## Highlights
+- Improved instrumentation with better configurability, Prometheus support and user-friendly features
+- Docker images are smaller and free of reported vulnerabilities
+- Improved WebSocket's stream management support
+- Erlang 27 compatibility
+- Various improvements and fixes
+
+## Added
+- Instrumentation (#4226, #4223)
+  - Modules:
+    - `mod_mam` (#4224, #4253, #4265, #4352)
+    - `mod_muc` (#4268)
+    - `mod_bosh` (#4281)
+    - `mod_csi` (#4284)
+    - `mod_event_pusher` (#4285)
+    - `mod_ping` (#4288)
+    - `mod_global_distrib` (#4292)
+    - `mod_pubsub` (#4297)
+    - `mod_websoctkets` (#4299)
+  - Hooks (#4289):
+     - Privacy (#4298)
+     - Authentication (#4301)
+     - Session (#4305)
+     - C2S (#4310)
+     - Roster (#4295)
+  - `async_pools` (#4337)
+  - `mongoose_wpool` (#4306)
+  - `mongoose_backend` (#4282)
+  - `mongoose_user_cache` (#4323)
+  - Router, S2S and component (#4319)
+  - Probes (#4272)
+  - Logging support (#4238)
+  - Dynamic handlers (#4247)
+  - C2S TCP/TLS listener (#4304)
+  - Authentication events (#4311)
+  - Session management (#4313)
+  - Global probes and counters (#4317)
+  - Documentation (#4355, #4358, #4360)
+  - Misc (#4228, #4249, #4287, #4312, #4324, #4335, #4363, #4227, #4368, #4296)
+- Improved stream management support for WebSockets (#4303)
```

Comment:
I would add it into the fixed section, as the goal of the PR was to fix the bug.